system_test/run.py

```diff
@@ -220,157 +222,157 @@ def test_basic_consuming(self):
+#
+#
+#         self.assert_s3_file_contents(pfx+'system-test-00001-000000000000.gz', expected_partitions[1], gzipped=True)
+#
+#         # partition 2 (new partition will start from offset 0)
+#         self.assert_s3_file_contents('systest/last_chunk_index.system-test-00002.txt',
+#                                      pfx+'system-test-00002-000000000000.index.json')
+#
+#         self.assert_s3_file_contents(pfx+'system-test-00002-000000000000.index.json',
+#                                     '{"chunks":[{"byte_length_uncompressed":1020,"num_records":34,"byte_length":139,"byte_offset":0,"first_record_offset":0}]}')
+#
+#
+#         self.assert_s3_file_contents(pfx+'system-test-00002-000000000000.gz', expected_partitions[2], gzipped=True)
+
+#     def test_binary_format(self):
+#         global g_producer
+#         topic = "binary-system-test"
+#
+#         try:
+#             # delete the topic before we start so we don't get extra messages
+#             sys.stderr.write(str(subprocess.check_output([os.path.join(this_dir, 'standalone-kafka/kafka/bin/kafka-topics.sh'),
+#                              '--bootstrap-server', 'localhost:9092', '--delete', '--topic', topic])))
+#         except subprocess.CalledProcessError:
+#             sys.stderr.write("topic does not exist")
+#
+#         debug=False
+#         s3connect = runS3ConnectStandalone(
+#             debug=debug,
+#             worker_props ='system-test-binary-worker.properties',
+#             sink_props ='system-test-s3-binary-sink.properties')
+#
+#         # messages produced asynchronously - synchronous producing makes it likely
+#         # they will be split into different flushes in connect
+#         expected_data = ''
+#         for i in range(0, 100):
+#             record = '{}\n'.format(i)
+#             g_producer.send(topic, bytes(record, encoding='utf8'))
+#             expected_data += record
+#
+#         g_producer.flush()
+#
+#         ok, line = dumpServerStdIO(s3connect, "Wait for binary sink to process and commit",
+#                                    until="Successfully uploaded chunk for binary-system-test-0",
+#                                    until_fail="ERROR",
+#                                    timeout=(1000000 if debug else 15), trim_indented=True)
+#
+#         self.assertTrue(ok, msg="Didn't get success message but did get: {}".format(line))
+#
+#         # run the reader to dump what we just wrote
+#         env = {
+#             'CLASSPATH': os.path.join(this_dir, 'build/libs/system_test-all.jar')
+#         }
+#         cmd = [os.path.join(this_dir, 'standalone-kafka/kafka/bin/kafka-run-class.sh'),
+#                "com.spredfast.kafka.connect.s3.source.S3FilesReader",
+#                os.path.join(this_dir, 'system-test-s3-binary-sink.properties')]
+#
+#         self.assertEqual(expected_data, subprocess.check_output(cmd, env=env))
```

Comment:
Should this be removed?

Reply:
Deletion of the entire `system_test` directory will occur as a part of https://sugarcrm.atlassian.net/browse/CXP-2417